webpack.config.js

```diff
@@ -23,51 +18,47 @@ module.exports = {
       {
         test: /\.css$/,
         use: [
-          MiniCssExtractPlugin.loader,
           'css-loader',
           'postcss-loader',
         ]
       },
       {
-        test: /\.ico$\.woff$|\.ttf$|\.wav$|\.mp3$/,
-        loader: 'file-loader'  // <-- retain original file name
```

Comment:
please don't use the `file-loader`, it's deprecated and not supported by HtmlBundlerPlugin